src/views/Timesheets/components/SavedShiftsList/index.js

```diff
@@ -0,0 +1,306 @@
+            <Grid item xs={3}>
+              <Typography variant="body2" style={styles.headerItem}>
+                JOB
+              </Typography>
+            </Grid>
+            <Grid item xs={2}>
+              <Typography variant="body2" style={styles.headerItem}>
+                TIME IN
+              </Typography>
+            </Grid>
+            <Grid item xs={2}>
+              <Typography variant="body2" style={styles.headerItem}>
+                TIME OUT
+              </Typography>
+            </Grid>
+            <Grid item xs={2}>
+              <Typography variant="body2" style={styles.headerItem}>
+                RATE
+              </Typography>
+            </Grid>
+            <Grid item xs={2}>
+              <Typography variant="body2" style={styles.headerItem}>
+                HOURS WORKED
+              </Typography>
+            </Grid>
+          </Grid>
+        </div>
+      </Grid>
+    );
+
+    let shiftsList = null;
+    shiftsList = this.props.shifts.map((shift, index) => (
+      <ShiftItem deleteShift={this.props.deleteShift} value={shift} key={index} />
+    ));
+
+    const supervisorDropdown = (
+      <FormControl
+        style={{
+          width: 252,
+        }}
+      >
+        <InputLabel>Submit To</InputLabel>
+        <Select
+          value={this.state.selectedSupervisor}
+          onChange={e => {
+            this.setState({
+              selectedSupervisor: e.target.value,
+            });
+          }}
+          input={<Input id="supervisor" />}
+        >
+          <MenuItem label="direct supervisor" value={this.state.directSupervisor}>
+            <Typography>
+              {this.state.directSupervisor !== null && this.state.directSupervisor.name}
+            </Typography>
+          </MenuItem>
+          <MenuItem label="reporting supervisor" value={this.state.reportingSupervisor}>
+            <Typography>
+              {this.state.reportingSupervisor !== null && this.state.reportingSupervisor.name}
+            </Typography>
+          </MenuItem>
+        </Select>
+      </FormControl>
+    );
+
+    let content = <></>;
+    if (this.props.shifts.length === null) {
+      content = (
+        <Card>
+          <CardContent>
+            <GordonLoader />
+          </CardContent>
+        </Card>
+      );
+    } else if (this.props.shifts.length > 0) {
+      content = (
+        <>
+          {confirmationBox}
+        <Card>
+          <CardContent>
+            <CardHeader title={cardTitle} />
+            <Grid
+              className="shift-list"
+              container
+              spacing={2}
+              justify="space-around"
+              alignItems="center"
+              alignContent="center"
+              style={styles.boxShadow}
+            >
+              {header}
+              {shiftsList}
+            </Grid>
+          </CardContent>
+          {cardTitle === "Submitted Shifts" &&
+              <CardContent>
```

Comment:
Should lines 239-248 and 251-260 be exactly the same?

Reply:
Good catch, we will Or the cardTitles to check if it is either Submitted Shifts or Approved Shifts and display the same content if that is true

Reply:
so basically something like 
```jsx
{(cardTitle === "Approved Shifts" || cardtitle === "Submitted Shifts") &&
    <CardContent>
      <Grid container>
        <Grid item xs={12} sm={6}>
          <Typography variant="h6">Total hours worked: {totalHoursWorked}</Typography>
        </Grid>
        <Grid item xs={12} sm={6}>
          <Typography variant="h6">Estimated gross pay: ${totalEstimatedPay}</Typography>
        </Grid>
      </Grid>
    </CardContent>
}
```